### ISSUE TITLE
爱可可微博热门分享(11.7)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#323]
No 1. <a href="https://weibo.com/1402400261/OF2ZNhR74">AI-Investigator：基于 Python 的自动化 AI 系统，旨在分析任何类型的网站内容，并使用 Claude 3.5 Sonnet API 和 Firecrawl 生成结构化报告</a>
No 2. <a href="https://weibo.com/1402400261/OEWQzx5h6">#转发赠书#携手@博文视点Broadview 送出3本《具身智能机器人系统》</a>
No 3. <a href="https://weibo.com/1402400261/OF2YKxSgl">Meme Search：一键索引表情包，让它们根据内容和文字快速被检索，为网络斗图增添乐趣</a>
No 4. <a href="https://weibo.com/1402400261/OEZXYoZPv">OuteTTS：一款实验性的文本转语音模型，利用纯语言模型进行文本到语音转换的创新方法，通过音频标记和强制对齐学习语音合成，让你可以用自己的声音生成文本语音</a>
No 5. <a href="https://weibo.com/1402400261/OFbRb03h7">Browser-Use：开源的网页自动化库，可以让大型语言模型（LLM）与网站互动，通过简单接口实现网页自动化操作，支持多标签管理、XPath提取和视觉模型处理</a>
No 6. <a href="https://weibo.com/1402400261/OFbQDAlF6">NVIDIA Cosmos Tokenizer：高效的图像和视频神经 tokenizer 套件，它提高了视觉 tokenization 的状态艺术，并为大型自回归变换器（如 LLMs）或扩散生成器的可扩展、健壮和高效开发铺平了道路</a>
No 7. <a href="https://weibo.com/1402400261/OF6c3vvpm">[LG]《Photon: Federated LLM Pre-Training》</a>
No 8. <a href="https://weibo.com/1402400261/OEZWP5Gas">Hunyuan-Large：腾讯推出的大型 MoE（Mixture of Experts）模型，该模型拥有 3890 亿参数和 520 亿激活参数，是业界目前最大的开源 Transformer 基础 MoE 模型，专注于自然语言处理和长文本理解</a>
No 9. <a href="https://weibo.com/1402400261/OEWQxt0g7">一直都很关注大语言模型及其应用，尤其在翻译领域。平时科研工作中会同时使用多个翻译工具，包括有道翻译、Google翻译等，不得不说真正把这个技术应用好的软件，真不多。大语言模型技术的核心优势就在于对语言的深刻理解和卓越的表达能力，也因此翻译地更加自然和准确。用下来发现有道翻译在专业文献 ......</a>
No 10. <a href="https://weibo.com/1402400261/OF30ru5HZ">几篇论文实现代码</a>
No 11. <a href="https://weibo.com/1402400261/OF697pJ7F">[LG]《ADOPT: Modified Adam Can Converge with Any β2β_2 with the Optimal Rate》</a>
No 12. <a href="https://weibo.com/1402400261/OF65Y03yM">[LG]《Discovering Data Structures: Nearest Neighbor Search and Beyond》</a>
No 13. <a href="https://weibo.com/1402400261/OF69c41nq">本文揭示了Adam优化算法收敛性问题的根本原因，并提出了一种新的自适应梯度方法ADOPT，该方法通过巧妙地去除当前梯度与二阶矩估计的相关性，在理论上保证了最优收敛速度，并在实践中展现出优越的性能和鲁棒性，尤其在高噪声环境下优势明显。</a>
No 14. <a href="https://weibo.com/1402400261/OFc8As17b">SylphAI-Inc LLM-engineer-handbook：大型语言模型资源手册，汇集模型训练、部署、微调和构建应用的精选资源，覆盖AI社区的热门领域</a>
No 15. <a href="https://weibo.com/1402400261/OEQef6Gao">LLaMA-O1：一个大型推理模型框架，用于 PyTorch 和 HuggingFace 的训练、推理和评估，集成了蒙特卡洛树搜索（MCTS）、自我对弈强化学习、PPO、AlphaGo Zero 的双策略范式和大型语言模型</a>
No 16. <a href="https://weibo.com/1402400261/OF5SbwGkm">早！[太阳] ​</a>
No 17. <a href="https://weibo.com/1402400261/OF2Ox4dla">[TAI科学]冯·诺依曼的机械心脏：人工生命的起源与未来</a>
No 18. <a href="https://weibo.com/1402400261/OF3nrvFbO">晚安～ [月亮] ​</a>
No 19. <a href="https://weibo.com/1402400261/OEQdKvu3v">机器人操控相关资源列表：汇集了关于机器人操控的论文、代码和相关网站的全面列表，是机器人研究者的宝贵资源</a>
No 20. <a href="https://weibo.com/1402400261/OEQu4z4KO">Claude Vision Object Detection：一款强大的Python图像识别工具，能够利用Claude 3.5 Sonnet Vision API检测并可视化图像中的对象，自动绘制边界框、标注对象并显示置信度分数</a>
No 21. <a href="https://weibo.com/1402400261/OEGKOd9tu">MiniMates：极速数字人算法，让你的桌面电脑也能拥有定制化的AI伙伴，支持语音和表情驱动</a>
No 22. <a href="https://weibo.com/1402400261/OF2U75sUJ">ComfyCanvas：一个与ComfyUI配合使用的画布工具，主要功能是让用户在ComfyUI界面中进行图像编辑和操作，具有多种按钮描述和快捷键，支持图像裁剪、比较、移动、擦除等多种操作</a>
No 23. <a href="https://weibo.com/1402400261/OF2lKa5xY">[TAI前沿]多层人工智能：用更多AI解决问题的艺术</a>
No 24. <a href="https://weibo.com/1402400261/OFbMtzayO">InkSight：通过学习阅读和书写实现离线到在线...</a>
No 25. <a href="https://weibo.com/1402400261/OF5TPwGTD">[CL]《SLED: Self Logits Evolution Decoding for Improving Factuality in Large Language Models》</a>
No 26. <a href="https://weibo.com/1402400261/OEQsOhYJP">askrepo：一款利用大型语言模型（LLM）阅读源代码的工具，能够理解代码内容并回答相关问题，特色在于与Google Gemini API的结合使用</a>
No 27. <a href="https://weibo.com/1402400261/OFc776q5W">Aide：开源人工智能原生代码编辑器，集成了强大的VS Code功能和先进的AI能力，提供聊天+编辑工作流、主动代理、内联编辑小部件、智能代码补全和AST导航</a>
No 28. <a href="https://weibo.com/1402400261/OEZX3w0SV">腾讯 Hunyuan3D-1：一款统一的文本到3D和图像到3D生成框架，能够快速生成3D模型，支持中文和英文文本提示，具有高速生成和良好泛化能力</a>
No 29. <a href="https://weibo.com/1402400261/OEGXWgQul">Cofounder：一款AI生成的应用，提供全栈解决方案和生成式用户界面，助你快速构建后端、数据库和具有状态的网络应用</a>
No 30. <a href="https://weibo.com/1402400261/ODVSS6iB1">PaperVision：自定义OpenCV算法的图形化节点编辑器，灵感来源于Blender和Unreal Engine的蓝图，提供直观的用户界面和实时预览功能</a>
No 31. <a href="https://weibo.com/1402400261/OEzCC9da4">AI Agent实战手册：一本深入浅出的AI开发指南，涵盖从理论基础到多个应用领域的实战案例，特色在于多智能体系统、可解释AI与伦理安全探讨</a>
No 32. <a href="https://weibo.com/1402400261/OEGZ0unWj">机器人学习资源库：汇集了关于仿人机器人学习的学术论文，主要按研究任务分类，特别偏好有真实机器人实验和开源代码的论文</a>
No 33. <a href="https://weibo.com/1402400261/OEWJGxUFi">[LG]《Gradient Methods with Online Scaling》</a>
No 34. <a href="https://weibo.com/1402400261/OEGJQhlyP">PDF Extract API：一个文档（PDF）提取和解析 API，使用先进的现代 OCR 技术和 Ollama 支持的模型，能匿名化文档、移除个人识别信息（PII），并将任何文档或图片转换为结构化的 JSON 或 Markdown 格式</a>
No 35. <a href="https://weibo.com/1402400261/OEGYe43gX">Someday：免费开源的Gmail用户日程安排工具，使用Google Apps Script搭建，提供类似Calendly的简单替代方案</a>
No 36. <a href="https://weibo.com/1402400261/OEH7XsFyt">pg_flo：实时数据流、转换和路由工具，用于PostgreSQL数据库，能够捕捉数据变更并进行处理</a>
No 37. <a href="https://weibo.com/1402400261/OEWHtfmzX">[LG]《Thinking Forward and Backward: Effective Backward Planning with Large Language Models》</a>
No 38. <a href="https://weibo.com/1402400261/OF6gOldd3">今日推介(第1582期)：基于自逻辑演化解码的大型语言模型事实性改进、大语言模型能在规定的痛苦和快乐状态之间做出权衡吗、数据结构的端到端学习、改进的Adam能以最优速率收敛于任意β₂ ......</a>
No 39. <a href="https://weibo.com/1402400261/ODFY9AmZY">x.infer：框架无关的计算机视觉推理库，它允许用户通过统一的 Python API 运行来自不同框架的多种模型的推理，支持超过1000+模型</a>
No 40. <a href="https://weibo.com/1402400261/ODVLV7MU9">TrustGraph Engine：一个不依赖特定大型语言模型（LLM）的知识Agent开发平台，提供工具、服务、图数据库和向量数据库，帮助部署可靠、可扩展、准确的AI代理，支持批量文档摄入、自动化知识图谱构建、自动化向量化、模型无关的LLM集成等</a>
No 41. <a href="https://weibo.com/1402400261/OEN5YfMqR">[CL]《Evolving Alignment via Asymmetric Self-Play》</a>
No 42. <a href="https://weibo.com/1402400261/OFbO2vzVD">[TAI快报]从事实性到联邦学习：五项研究引领大型语言模型新方向</a>
No 43. <a href="https://weibo.com/1402400261/OEQeYt66w">Run:ai Model Streamer：用于流式传输张量文件到 GPU 内存的 Python SDK，它支持并发和流式加载，并提供了一个 API 来简化 AI 模型的构建和加载过程</a>
No 44. <a href="https://weibo.com/1402400261/OEAfCvS4E">工业异常检测大全：汇集多种工业异常检测方法和数据集的资源库，用于帮助研究人员和工程师在工业领域快速找到合适的异常检测技术和数据资源</a>
No 45. <a href="https://weibo.com/1402400261/OEH6TwJdz">pqoqubbw icons：精心制作的动画图标库，提供美观且实用的图标资源，适用于网站和应用界面设计，特色在于其动画效果</a>
No 46. <a href="https://weibo.com/1402400261/ODVMn3fSb">SimpleMind：为人类设计的AI工具库，旨在简化Python中AI API的使用体验，强调易用性、以人为中心的设计和最少的配置需求，让与AI的互动就像友好对话一样简单</a>
No 47. <a href="https://weibo.com/1402400261/OF6c8aTId">Photon 提出了一种创新的联邦式大型语言模型预训练系统，通过巧妙地结合小批量高学习率策略和高效的通信机制，在低带宽环境下实现了比集中式训练更优的性能和效率，为大规模语言模型的训练开辟了新的可能性。//[LG]《Photon: Federated LLM Pre-Training》</a>
No 48. <a href="https://weibo.com/1402400261/NkMCDnmka">ChatGPT Plugin for Zotero：Zotero的ChatGPT插件，可以方便地访问概念解释、内容摘要和双向翻译功能</a>
No 49. <a href="https://weibo.com/1402400261/OE2erdGqj">#免费##抽奖# 携手@博文视点Broadview 送出3本《事理图谱：概念与技术》</a>
No 50. <a href="https://weibo.com/1402400261/OblFsztVE">tiny-gpu：开源的 Verilog 语言编写的 GPU 设计项目，旨在帮助人们从底层上理解 GPU 是如何工作的</a>
